packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -184,7 +189,7 @@ const typeBrandHint = `This field is a type brand and is not populated at runtim
 https://github.com/grpc/grpc-node/pull/2281`;
 
 function formatTypeBrand(formatter: TextFormatter, messageType: Protobuf.Type) {
-  formatComment(formatter, typeBrandHint);
+  formatComment(formatter, typeBrandHint, messageType.options);
```

Comment:
I don't think this one is right. This comment isn't on the message interface itself, but rather on a synthetic field in the interface. So it doesn't make sense for this field to inherit the message type's deprecated status.

Reply:
@murgatroid99 I removed the options argument for this format call.